File: ncarg2d/src/libncarg/ngmisc/ngckop.f
C
C	$Id: ngckop.f,v 1.4 2008-07-27 00:17:17 haley Exp $
C                                                                      
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      INTEGER FUNCTION NGCKOP(IWKID)
C
C  Function to check if the workstation, whose workstation ID is
C  given in IWKID, is currently open.  If it is open, then the
C  function returns a 1, otherwise the function returns a 0.
C
      CALL GQOPWK(0,IER,NOPN,LID)
      IF (NOPN .EQ. 0) THEN
        NGCKOP = 0
        RETURN
      ELSE
        DO 10 I=1,NOPN
          CALL GQOPWK(I,IER,NOPN,LID)
          IF (IWKID .EQ. LID) THEN
            NGCKOP = 1
            RETURN
          ENDIF
   10   CONTINUE
      ENDIF
C
      NGCKOP = 0
      RETURN
      END

File: ncarg2d/man/ngmisc/ngckop.m
.TH NGCKOP 3NCARG "October 1996" UNIX "NCAR GRAPHICS"
.na
.nh
.SH NAME
NGCKOP - a function to check if a specified GKS workstation is open.
.SH SYNOPSIS
INTEGER FUNCTION NGCKOP(WKID)
.SH C-BINDING SYNOPSIS
#include <ncarg/ncargC.h>
.sp
int c_ngckop(int wkid)
.SH DESCRIPTION 
.IP WKID 12
(an input variable of type INTEGER) specifying a  GKS workstation
identifier.
.SH C-BINDING DESCRIPTION
The C binding argument descriptions are the same as the FORTRAN
argument descriptions.
.SH USAGE
The function returns a "1" if the workstation identified by
WKID is open, otherwise it returns a "0".
.SH EXAMPLES
.nf

        IOPEN = NGCKOP(WKID)

.fi
sets IOPEN to "1" if WKID is open and sets IOPEN to "0" otherwise.
.SH ACCESS
To use NGCKOP or c_ngckop, load the NCAR Graphics libraries ncarg, ncarg_gks,
and ncarg_c, preferably in that order.  
.SH SEE ALSO
Online URL: http://ngwww.ucar.edu/ngdoc/ng/gks/gkshome.html
.SH COPYRIGHT
Copyright (C) 1987-2009
.br
University Corporation for Atmospheric Research
.br
The use of this Software is governed by a License Agreement.
